README.md

```diff
@@ -0,0 +1,87 @@
+
+<!-- README.md is generated from README.Rmd. Please edit that file -->
+
+# hydromad
+
+<!-- badges: start -->
+
+<!-- badges: end -->
+
+The goal of hydromad is to provide a modelling framework for
+environmental hydrology: water balance accounting and flow routing in
+spatially aggregated catchments.
+
+Hydromad supports simulation, estimation, assessment and visualisation
+of flow response to time series of rainfall and other drivers. A minimal
+unit hydrograph framework is used, where areal rainfall is passed
+through a soil moisture accounting (SMA) model to estimate effective
+rainfall; this is then passed through a routing model to estimate
+streamflow. Included are several implementations of common hydrological
+models consistent with this framework.
+
+## Installation
+
+<!--- You can install the released version of hydromad from [CRAN](https://CRAN.R-project.org) with:
+
+``` r
+install.packages("hydromad")
+```
+--->
+
+Currently you can install the development version from
+[GitHub](https://github.com/) with:
+
+``` r
+# install.packages("devtools")
+devtools::install_github("JosephGuillaume/hydromad")
+```
+
```

Comment:
These instructions don't match those on the website. 
http://hydromad.catchment.org/#installation
https://github.com/josephguillaume/hydromad/blob/master/web/template.html#L140
I think we still want to encourage installation of the compiled version, which doesn't require installation of Rtools

I think what we want is actually to simplify the instructions of the website and here to add hydromad.catchment.org to repos, i.e. `options(repos = r)`. That'll make it easier to deal with dependencies.

Reply:
OK I will update to the same as the website and also make the link to the website more explicit (I forgot that)
Do we want people to install from github at all? 

"here to add hydromad.catchment.org to repos": I am not sure I know how to do that